js/components/user/User.js

```diff
@@ -191,27 +212,28 @@ class User extends React.Component {
                     <div className='row'>
                         <div className='col-12 col-sm-6'>
                             <HorizontalInput type='text' name='firstName' label={`${this.i18n('user.first-name')}*`}
-                                             disabled={currentUser.role !== ROLE.ADMIN && currentUser.username !== user.username}
+                                             disabled={isUsingOidcAuth()}
                                              value={user.firstName} labelWidth={3} inputWidth={8}
                                              onChange={this._onChange}/>
                         </div>
                         <div className='col-12 col-sm-6'>
                             <HorizontalInput type='text' name='lastName' label={`${this.i18n('user.last-name')}*`}
-                                             disabled={currentUser.role !== ROLE.ADMIN && currentUser.username !== user.username}
+                                             disabled={isUsingOidcAuth()}
                                              value={user.lastName} labelWidth={3} inputWidth={8}
                                              onChange={this._onChange}/>
                         </div>
                     </div>
                     <div className='row'>
                         <div className='col-12 col-sm-6'>
                             <HorizontalInput type='text' name='username' label={`${this.i18n('user.username')}*`}
-                                             disabled={!user.isNew} labelWidth={3} inputWidth={8}
+                                             disabled={!user.isNew || isUsingOidcAuth()}
+                                             labelWidth={3} inputWidth={8}
                                              value={user.username} onChange={this._onChange}
                                              iconRight={user.isNew ? generateButton : null}/>
                         </div>
                         <div className='col-12 col-sm-6'>
                             <HorizontalInput type='email' name='emailAddress' label={`${this.i18n('users.email')}*`}
-                                             disabled={currentUser.role !== ROLE.ADMIN && currentUser.username !== user.username}
+                                             disabled={isUsingOidcAuth()}
```

Comment:
@LaChope  please fix
This will not work in internal-auth, if you are a regular user that is looking at different user.

![image](https://github.com/kbss-cvut/record-manager-ui/assets/3962237/6250b044-ef62-4f2e-a2ca-5f78ac6ed219)

Reply:
Resolved with c0fe1368f5cb11269b3f56863ba910e878294afc